HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -2267,6 +2268,33 @@ def self.apply_hvac(runner, hpxml_bldg, weather, convert_shared_systems, unit_nu
       end
     end
 
+    # EER
+    (hpxml_bldg.cooling_systems + hpxml_bldg.heat_pumps).each do |hvac_system|
+      if hvac_system.is_a?(HPXML::CoolingSystem)
+        next unless [HPXML::HVACTypeCentralAirConditioner,
+                     HPXML::HVACTypeMiniSplitAirConditioner].include? hvac_system.cooling_system_type
+      elsif hvac_system.is_a?(HPXML::HeatPump)
+        next unless [HPXML::HVACTypeHeatPumpAirToAir,
+                     HPXML::HVACTypeHeatPumpMiniSplit].include? hvac_system.heat_pump_type
+      end
+      next unless hvac_system.cooling_efficiency_eer.nil? && hvac_system.cooling_efficiency_eer2.nil?
+
+      seer = hvac_system.cooling_efficiency_seer
+      case hvac_system.compressor_type
+      when HPXML::HVACCompressorTypeSingleStage
+        # FIXME: Review compared to AHRI data
+        cop = 0.2692 * seer + 0.2706 # Regression based on inverse model
+      when HPXML::HVACCompressorTypeTwoStage
+        # FIXME: Review compared to AHRI data
+        cop = 0.2773 * seer - 0.0018 # Regression based on inverse model
+      when HPXML::HVACCompressorTypeVariableSpeed
+        # FIXME: Need to develop something here based on AHRI/NEEP data
+        cop = 4.5
+      end
```

Comment:
Default EER if we don't have an EER/EER2 input. The 1-speed and 2-speed defaults are pulled from hvac.rb. The variable-speed default is just hardcoded for now because it's not currently used, though it will start being used when we adopt the RESNET equations, so we'll need to develop a better default here.